ArcadiaParties.CQRS/Handlers/GetCurrentUserHandler.cs

```diff
@@ -2,12 +2,13 @@
 using ArcadiaParties.Data.Abstractions.DTOs;
 using ArcadiaParties.Data.Abstractions.Repositories;
 using MediatR;
+using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace ArcadiaParties.CQRS.Handlers
 {
-    internal class GetCurrentUserHandler : IRequestHandler<GetCurrentUserQuery, UserDTO>
+    internal class GetCurrentUserHandler : IRequestHandler<GetCurrentUserQuery, IEnumerable<string>>
```

Comment:
Did you hurry somewhere when changed this? Why is it still named `GetCurrentUserHandler` if it returns just roles?

Reply:
Renamed